waldronlab/bioconductor.yaml

```diff
@@ -1,12 +1,15 @@
 manifest:
-  name: waldronlab
+  name: bioconductor
   version: 1.0.0
   commands:
-  - command: rdevel
+  - command: rd
     docker_image: bioconductor/bioconductor_full:devel
     docker_command: R
-    dockerargs: "-u bioc -i -v ${HOME}/R/bioc-devel:/home/bioc/R/library -v ${HOME}:/home/bioc"
-  - command: bgit
+    dockerargs: "-t -i --user bioc"
+  - command: bashd
```

Comment:
Is your `bashd` command accomplishing the same thing I was thinking about here? 

https://github.com/databio/bulker/issues/8

if so, this is an interesting solution... you'd have to do it manually for every container, though. Eventually I'll come up with a way to make it systematic so you can just immediately open a shell into any of your manifest-listed images...

Reply:
I just made it possible to call `_command` to enter a shell for that container (but requires the dev branch of bulker)

Reply:
Yes, it accomplishes the same thing - I just wanted to enter a bash shell inside the container, for interactive debugging. So with the devel version I could run, e.g., `_rd` instead of creating this `dbash` command. Good idea.